src/proto/git_fs.proto

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package bowler.git_fs;
+
+option java_multiple_files = true;
+option java_package = "com.commonwealthrobotics.proto.gitfs";
+option java_outer_classname = "GitFSOuterClass";
```

Comment:
Does this cause the generated class name to be `GitFSOuterClass`?

Reply:
Yes, for some of the boilerplate. This matches the same pattern that's autogenerated for the other ones. The reason I have to manually specify it here is because otherwise it tries to use `GitFs` (converting the filename from snake_case to PascalCase), which then produces a warning because it only differs from the service name `GitFS` by capitalization.

Reply:
Damn, I'm not sure what I'd rather do. What do you think?

Reply:
Well, the reason I originally had it as `git_f_s` was to make the PascalCase what we expect it to be, which avoided this issue. So we could go back to that. (Note that the `git_f_s` name would only show up on the protobuf side, and not in any of the Java API)

Reply:
I'll trust your judgement here. Go ahead and change it back. Sorry about this; I didn't realize this would be an issue.